docs/core/testing/unit-testing-with-dotnet-test.md

```diff
@@ -69,3 +102,13 @@ Since this mode is specifically designed for Microsoft.Testing.Platform, neither
 >
 > [!IMPORTANT]
 > If your test project supports VSTest but does not support MTP, an error will be generated.
+>
+> [!TIP]
+> You can customize the command-line arguments of your test application via `TestingPlatformCommandLineArguments` MSBuild property:
```

Comment:
This seems to be repeated from line 57.

Reply:
@gewarren Yes. It's intentional. One is under the VSTest mode of `dotnet test` and the other is under the MTP mode of `dotnet test`. The property is supported in both cases.